source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -1198,6 +1198,31 @@ TEST(NiFakeServiceTests, NiFakeService_GetAttributeViString_CallsGetAttributeViS
   EXPECT_THAT(response.attribute_value(), ElementsAreArray(attribute_char_array, expected_size));
 }
 
+TEST(NiFakeServiceTests, NiFakeService_AcceptViUInt32Array_CallsAcceptViUInt32Array)
+{
+  nidevice_grpc::SessionRepository session_repository;
+  std::uint32_t session_id = create_session(session_repository, kTestViSession);
+  NiFakeMockLibrary library;
+  nifake_grpc::NiFakeService service(&library, &session_repository);
+  const unsigned int uInt32Array[] = {1, 2, 3, 4, 5};
```

Comment:
Add atleast one element which uses full 32 bit length, so that we can make sure both MIN and MAX values work proprely.

Reply:
added large elements near max range of unsigned long which is ViUInt32

Reply:
Oh, I meant add lower as well as higher bounds. So you can have 0, 1, 2, 4294967294, 4294967295.

Also why did you change from `unsigned int` to `unsigned long`? Is `unsigned long` the actual range supported by ViUInt32?

Reply:
yes ViUint32 is converted to `unsigned long`
`typedef unsigned long       ViUInt32;`